.env_example

```diff
@@ -10,6 +10,4 @@ export APP_ENV=local
 
 export PORT=3000
 
-export SDR_USER=sdrUser
 export SDR_PASS=sdrPass
-
```

Comment:
This .env_example is never used unless copied to .env and, if all goes to plan, this file might disappear.  It does no harm to leave this file as it is.